tools/segmetrics/segmetrics.xml

```diff
@@ -0,0 +1,96 @@
+            <param name="hsd_sym" type="boolean" checked="false" truevalue="-measure-hsd_sym" falsevalue="" label="Hausdorff distance (symmetric)" />
+            <param name="hsd_e2a" type="boolean" checked="false" truevalue="-measure-hsd_e2a" falsevalue="" label="Hausdorff distance (ground truth to segmented)" />
+            <param name="hsd_a2e" type="boolean" checked="false" truevalue="-measure-hsd_a2e" falsevalue="" label="Hausdorff distance (segmented to ground truth)" />
+            <param name="nsd"     type="boolean" checked="false" truevalue="-measure-nsd"     falsevalue="" label="Normalized sum of distances" />
+            <param name="o_hsd_sym" type="boolean" checked="true"  truevalue="-measure-o_hsd_sym" falsevalue="" label="Object-based Hausdorff distance (symmetric)" />
+            <param name="o_hsd_e2a" type="boolean" checked="false" truevalue="-measure-o_hsd_e2a" falsevalue="" label="Object-based Hausdorff distance (ground truth to segmented)" />
+            <param name="o_hsd_a2e" type="boolean" checked="false" truevalue="-measure-o_hsd_a2e" falsevalue="" label="Object-based Hausdorff distance (segmented to ground truth)" />
+            <param name="o_nsd"     type="boolean" checked="true"  truevalue="-measure-o_nsd"     falsevalue="" label="Object-based normalized sum of distances" />
+            <param name="fs" type="boolean" checked="true" truevalue="-measure-fs" falsevalue="" label="Falsely split objects per image" />
+            <param name="fm" type="boolean" checked="true" truevalue="-measure-fm" falsevalue="" label="Falsely merged objects per image" />
+            <param name="fp" type="boolean" checked="true" truevalue="-measure-fp" falsevalue="" label="Spurious objects per image" />
+            <param name="fn" type="boolean" checked="true" truevalue="-measure-fn" falsevalue="" label="Missing objects per image" />
+        </section>
+
+    </inputs>
+    <outputs>
+       <data format="csv" name="results" from_work_dir="results.csv" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input_seg" value="input2.png"/>
+            <param name="input_gt"  value="input1.png"/>
+            <output name="results" value="results1.csv" ftype="csv" compare="diff"/>
+            <param name="is_seg_unique" value="True"/>
+            <param name="is_gt_unique"  value="True"/>
+        </test>
+        <test>
+            <param name="input_seg" value="input1.zip"/>
+            <param name="input_gt"  value="input2.zip"/>
+            <output name="results" value="results2.csv" ftype="csv" compare="diff"/>
+            <param name="is_seg_unique" value="True"/>
+            <param name="is_gt_unique"  value="True"/>
+            <param name="unzip" value="True"/>
+        </test>
+    </tests>
+    <help>
+        Image segmentation and object detection performance measures for 2-D image data.

```

Comment:
It would be nice to extend the help a bit. The more the merrier to help users :)

Reply:
yes, I did so in 2fe567aab47d7238760a8937496fc5cda36149c5 :)

Reply:
ace!